### DISCUSSION TITLE
Apache for players

### DISCUSSION THREAD

**Discussion by DevastatorCM** [Khopa/dcs_liberation#2108] [Q&A] [answered]
when the new update comes, is there a way to convert existing apache's into flyable ones instead of keeping them AI units? Maybe the squadrons can be duplicated somehow on existing missions which are converted from 5.1 into 5.2 to keep everybody happy?

**Comment by Starfire13** [comment, marked as answer]
Factions that can fly the Apache will have access to both Apaches.
